ONX.CRM.BLL/Services/ManagerService.cs

```diff
@@ -53,5 +53,22 @@ public Task<IEnumerable<Manager>> FindByUserIdAsync(string userId)
         {
             return _managerRepository.FindByUserIdAsync(userId);
         }
+
+        public void SaveImage(int id, byte[] content)
```

Comment:
Why you need this method? As far as I see, it works exactly the same as Update()